code/game/machinery/computer/atmos_alert.dm

```diff
@@ -14,7 +14,7 @@ var/global/list/minor_air_alarms = list()
 
 /obj/machinery/computer/atmos_alert/Initialize()
 	. = ..()
-	atmosphere_alarm.register_alarm(src, /obj/machinery/computer/station_alert/update_icon)
```

Comment:
as with previous event change, this needs explanation. Why is it being changed to a different format than the previous one? why not...

```atmosphere_alarm.register_alarm(src, .proc/update_icon)```